worldengine/basic_map_operations.py

```diff
@@ -1,9 +1,12 @@
 import math
-import random
+import numpy
 
 
-def random_point(width, height):
-    return random.randrange(0, width), random.randrange(0, height)
+def random_point(width, height, randint_substitute=None):
```

Comment:
By the way: The method currently seems to be called only at a single point (outside from the test-suite): world.py, ~line 335
Maybe we could avoid the whole discussion by just completely removing it and "inlining" the code instead. Or by removing the default value of "None" and enforcing the usage of the last parameter to make the caller fully aware of what he is doing.